nebula/ui/components/VPNPanel.qml

```diff
@@ -47,19 +46,6 @@ Item {
                 sourceSize.height: logoSize
                 sourceSize.width: logoSize
                 fillMode: Image.PreserveAspectFit
-                layer.enabled: true
-
-                Rectangle {
-                    id: mask
-
-                    anchors.fill: parent
-                    radius: logoSize / 2
-                    visible: false
-                }
-
-                layer.effect: VPNOpacityMask {
-                    maskSource: maskImage ? mask : undefined
-                }
```

Comment:
If I’m not mistaken this mask is supposed to only be used for `VPNAvatar`. We already have a mask for the avatar in that component, but it did not work as expected. Fixed the mask there so this one should be redundant.